ios/Actors/FlowStarter.swift

```diff
@@ -12,4 +12,9 @@ class FlowStarter: NSObject {
     func start(from viewController: UIViewController) {
         Fidel.start(from: viewController)
     }
+    
+    func verifyCard(from viewController: UIViewController, id: String, consentID: String, last4Digits: String) {
```

Comment:
I think this FlowStarter class can just be removed. Not sure why it exists. It just forwards calls to Fidel functions. My bad about introducing it. I was probably tired or something. When I initially worked on this library as a contractor, nobody reviewed my code, so it's probably a mistake.